test/runtests.jl

```diff
@@ -23,11 +23,15 @@ my_tests = ["ambiguity.jl",
             "contrasts.jl",
             "extension.jl",
             "traits.jl",
-            "protect.jl"]
+            "protect.jl",
+            "vif.jl"]
 
 @testset "StatsModels" begin
     @testset "aqua" begin
-        Aqua.test_all(StatsModels; ambiguities=false)
+        # because VIF and GVIF are defined in StatsAPI for RegressionModel,
+        # which is also defined there, it's flagged as piracy. But
+        # we're the offical implementers so it's privateering.
+        Aqua.test_all(StatsModels; ambiguities=false, piracy=false)
```

Comment:
Looks like you can pass something like `ambiguities=(exclude=[vif, gvif],)`.

Reply:
That works for `ambiguities` but not `piracy` :frowning:

Reply:
What about `piracy=(treat_as_own=[vif, gvif],)` or `piracy=(treat_as_own=[RegressionModel],)`? The Aqua manual seems tailored to this PR: :-D
> treat_as_own = Union{Function, Type}[]: The types in this container are considered to be "owned" by the module m. This is useful for testing packages that deliberately commit some type piracy, e.g. modules adding higher-level functionality to a lightweight C-wrapper, or packages that are extending StatsAPI.jl.

Reply:
merci! updated